fraqbot/tests/test_newjob.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+import threading
+
+from Legobot.Lego import Lego
+from mock import patch
+
+LOCAL_PATH = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    '..',
+    'Local'
+)
+sys.path.append(LOCAL_PATH)
+from newjob import NewJob  # noqa: E402
+
+LOCK = threading.Lock()
+BASEPLATE = Lego.start(None, LOCK)
+LEGO = NewJob(BASEPLATE, LOCK)
+
+
+def test_listening_for():
+    # !job
+    assert LEGO.listening_for({'text': 'job'}) is False
+    assert LEGO.listening_for({'text': '!job moin'}) is True
+    assert LEGO.listening_for({'text': '!job'}) is True
+    assert LEGO.listening_for({'text': '!Job'}) is False
+    # !newjob
+    assert LEGO.listening_for({'text': 'newjob'}) is False
+    assert LEGO.listening_for({'text': '!newjob moin'}) is True
+    assert LEGO.listening_for({'text': '!newjob'}) is True
+    assert LEGO.listening_for({'text': '!Newjob'}) is False
+
+
+def test_get_job():
```

Comment:
These tests don't make any sense. _get_job returns a string, so we shouldn't be checking for True

Reply:
It's wrapped in `bool` . Just making sure it returns something, but yeah `instance of string ` would be better :)

Reply:
updated

Reply:
You can remove the `is True` from these asserts
isinstance returns a bool

Reply:
Ah true